### DISCUSSION TITLE
How can I scroll in a non-focused window?

### DISCUSSION THREAD

**Discussion by otakutyrant** [hyprwm/Hyprland#2590] [Q&A] [answered]
I use i3 and it has a fantastic feature: Users can move mouse cursor to a non-focused window like Google Chrome, and scroll the page in it immediately without changing focus. It is very useful when you code and read documents in the same time, avoiding adamant focus movement interrupts flow.

No matter whether it is a bug for Hyprland. I hope Hyprland allows this behavior, at least optional.

I know `input { follow_mouse = 1 }` can achieve this effect. However the i3 feature does not rely on the correspond config, neither I like it too.

**Comment by MightyPlaza** [comment, marked as answer]
follow_mouse = 2 and works for me

**Reply by otakutyrant** [reply]
Thank you!

**Reply by chaush-server** [reply]
GOD BLESS
